### PR TITLE
Add autoscale RU support for azure-spring-data-cosmos

### LINKED ISSUE #12711: Autoscaling using Spring-Data-Cosmos
https://github.com/microsoft/spring-data-cosmosdb/issues/545

### PR DESCRIPTION
**Pull request by johnmannix**
Resolves #12711

### CODE REVIEW

**Review by mbhaskar — APPROVED** [review]
@johnmannix  Thank you for the contribution. Looks good to me

File: sdk/cosmos/azure-spring-data-cosmos-test/src/test/java/com/azure/spring/data/cosmos/domain/AutoScaleSample.java [hunk 1]
```diff
@@ -0,0 +1,25 @@
+package com.azure.spring.data.cosmos.domain;
```
Comment (kushagraThapar) [inline sdk/cosmos/azure-spring-data-cosmos-test/src/test/java/com/azure/spring/data/cosmos/domain/AutoScaleSample.java:1]: Please add new file license.

**Review by kushagraThapar — CHANGES_REQUESTED** [review]
Overall looks good to me, except for one small change.

File: sdk/cosmos/azure-spring-data-cosmos-test/src/test/java/com/azure/spring/data/cosmos/domain/AutoScaleSample.java (hunk 1 above)
Comment (kushagraThapar) [reply sdk/cosmos/azure-spring-data-cosmos-test/src/test/java/com/azure/spring/data/cosmos/domain/AutoScaleSample.java:3]: Done.

**Review by kushagraThapar — APPROVED** [review]

**Pull request merged**